Foundations/JS/index.js

```diff
@@ -0,0 +1,94 @@
+function computerPlay(){
```

Comment:
use `prettier` extension in vscode to format these files

spacing is not there, looks ugly